challenge.rb

```diff
@@ -22,7 +26,11 @@ def is_of_age? age, minimum_age
 # Remember: `p`, `print`, and `puts` don't return their arguments.
 
 def okay_to_drink? age, drinking_age
-  # Replace this comment with your code.
+  if (is_of_age? age, drinking_age) == true
+      return "Come on in."
+  else
+      return "I'm sorry, #{age} isn't old enough to enter. The minimum age is #{drinking_age}."
+  end
```

Comment:
All correct! However, it could be a little tighter. For instance, the pattern `x == true` is redundant, since the expression is only ever true when `x` is true, and only ever false when `x` is false. In your code above, you could have just written
```
    if (is_of_age? age, drinking_age)
```